lib/instance-validator.js

```diff
@@ -112,7 +112,7 @@ class InstanceValidator {
           .then(newError => { throw newError || error; }))
       )
       .then(() => runHooks('afterValidate', this.modelInstance, this.options))
-      .return(this.modelInstance);
+      .return();
```

Comment:
It depends on what you mean by "breaking change". Yes, if someone depended on this behavior, then a new version would break their implementation, but resolving into the model instance is undocumented behavior.